micropub.php

```diff
@@ -525,25 +556,25 @@ public static function generate_post_content( $args ) {
 		// https://github.com/snarfed/wordpress-micropub/issues/56#issuecomment-300805891
 		$checkin = $props['checkin'][0];
 		if ( $checkin ) {
-			$name = $checkin['properties']['name'][0];
-			$urls = $checkin['properties']['url'];
+			$name    = $checkin['properties']['name'][0];
+			$urls    = $checkin['properties']['url'];
 			$lines[] = '<p>Checked into <a class="h-card p-location" href="' .
-				($urls[1] ?: $urls[0]) . '">' . $name . '</a>.</p>';
+			( $urls[1] ?: $urls[0] ) . '">' . $name . '</a>.</p>';
```

Comment:
wow this is *very* misleading. implies that it's a standalone statement, but it's really part of the line above. we need to loosen PHPCS to prevent changes like this.

Reply:
I was able to get some of these reversed. It looks like, when it realigned the whole page, it removed it. I'm manually adjusting and it isn't flagging.

Reply:
perfect. thank you!